apps/examples/st_things/light/sample_light_user.c

```diff
@@ -26,6 +26,10 @@
 
 #define TAG	"LIGHT_USER"
 
+#define SEC_ATTRIBUTE_PUSH_MESG_SOURCE		"x.org.iotivity.ns.source"
```

Comment:
As this is an open source code, it would be better to remove the prefix 'SEC' from all macro definitions.
Ex: From SEC_ATTRIBUTE_PUSH_MESG_SOURCE to PUSH_MESG_SOURCE_ATTRIBUTE